utils/keys.go

```diff
@@ -97,6 +92,8 @@ func ExportKeys(to io.Writer, s Exporter, from string) error {
 // header. If the file already exists, the file is truncated. Multiple
 // adjacent PEMs with the same "path" header are appended together.
 func ImportKeys(from io.Reader, to []Importer, fallbackRole string, fallbackGun string, passRet notary.PassRetriever) error {
+	// here is a small flowchart I made to clear up my understand while writing the cases in this function
+	// it is very rough, but it may help while reading this piece of code http://imgur.com/HQICWeO
```

Comment:
Sure, I'd be happy to but the only issue is that we would then need to maintain it if it's in the repo. Totally your call